site_scons/site_tools/extra/extra.py

```diff
@@ -93,15 +89,13 @@ def preprocess_emitter(source, target, env):
 def generate(env):
     """Setup the our custom tools"""
 
-    indent = find_indent()
+    indent = _find_indent()
 
-    generator = lambda source, target, env, for_signature: pp_gen(source,
-                                                                  target,
-                                                                  env, indent)
+    generator = lambda source, target, env, for_signature: _pp_gen(source, target, env, indent)
```

Comment:
nitpicking, but seems more pythonic like this for me
```suggestion
    generator = lambda source, target, env, _: _pp_gen(source, target, env, indent)
```

Reply:
Tried this, and it failed with python errors.